src/cmd/manage_nsfs.js

```diff
@@ -758,6 +780,35 @@ async function validate_account_args(data, action) {
     }
 }
 
+/**
+ * We take an object that was stringify
+ * (it unwraps ths sensitive strings, creation_date to string and removes undefined parameters)
+ * for validating against the schema we need an object, hence we parse it back to object
+ * @param {string} data
+ */
+function validate_account_schema_wrapper(data) {
+    try {
+        nsfs_schema_utils.validate_account_schema(JSON.parse(data));
+    } catch (err) {
+        dbg.error('validate_account_schema_wrapper: schema validation error:', err);
+        throw_cli_error(ManageCLIError.InvalidSchema, err.message);
+    }
+}
+
+/**
+ * We take an object that was stringify
+ * (it unwraps ths sensitive strings, creation_date to string and removes undefined parameters)
+ * for validating against the schema we need an object, hence we parse it back to object
+ * @param {string} data
+ */
+function validate_bucket_schema_wrapper(data) {
+    try {
+        nsfs_schema_utils.validate_bucket_schema(JSON.parse(data));
```

Comment:
try to avoid stringify and then parsing again, you can use the initial value

Reply:
Don't forget what I earned from the `JSON.stringify` that is not in the initial value - it removes `undefined` values and `unwarp` `SensitiveString` properties (either using it or doing it my own as I did at first). I added a comment to explain this.